uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -271,14 +271,16 @@ define([
 
       /**
        * Checks if given value is added for leave request list of document value ie., field required_documents
-       *  otherwise add it to list of required documents
+       *  otherwise add it to list of required documents (list is actually string of comma separated values for now)
        *
        * @param {String} value
        */
       vm.updateRequiredDocuments = function (value) {
-        var index = vm.leaveRequest.required_documents.indexOf(value);
+        var docsArray = vm.leaveRequest.required_documents ? vm.leaveRequest.required_documents.split(','): [];
```

Comment:
Again, this should *not* happen. The initialization should happen only in the `SicknessRequestInstance.init()` method, it shouldn't be the concern of the controller